quiz.js

```diff
@@ -0,0 +1,44 @@
+var grow= [];
+var branchNum= document.getElementById("branches");
+var leafNum= document.getElementById("leaf");
+var button = document.getElementById("button");
+
+function tree(branches, leaf) {
+	if (branchNum.value==="" || leafNum.value===""){
```

Comment:
Here you're making another read call to the DOM to grab the values of these input fields, but you already have those values, passed in as ```branches``` and ```leaf```